src/sidebar/components/sidebar-panel.js

```diff
@@ -48,14 +48,11 @@ function SidebarPanel({ children, panelName, title, onActiveChanged }) {
           <div className="sidebar-panel__title u-stretch">{title}</div>
           <div>
             <button
-              className="sidebar-panel__close-btn"
+              className="sidebar-panel__close-btn action-button--compact"
               onClick={closePanel}
               aria-label="close panel"
             >
-              <SvgIcon
-                name="cancel"
-                className="sidebar-panel__close-btn-icon"
-              />
+              <SvgIcon name="cancel" className="action-button--compact__icon" />
```

Comment:
This class doesn't appear to be defined anywhere in this PR.

Reply:
Ah! This happened because: `.action-button--compact` includes `.action-button`; `.action-button` has a `&__icon` rule. I'm not clear on if there's another classname I could use here to pick up this rule. Help? (see https://github.com/hypothesis/client/pull/1505/files#diff-c3589fe6c368ae77717f73e858de1ec8R7 )

Reply:
AFAIK the conventional way to handle that would be to make the `action-button--compact` class only include the customizations needed for a button. The markup would include both the base style and the variant. So the button here would have `<button className="action-button action-button--compact">`. See examples under the "Modifier" section at http://getbem.com/naming/. I suspect there are quite a few places in the existing markup where we're not doing this correctly.

Reply:
OK...how about now?